src/api/index.coffee

```diff
@@ -24,7 +24,7 @@ app.use cors()
 app.use helpers
 app.use bodyParser.json limit:'5mb', extended: true
 app.use bodyParser.urlencoded limit:'5mb', extended: true
-app.use logger
+app.use(httpLogger)
```

Comment:
Sorry, this is in `CoffeeScript`, but fortunately this isn't really any different than it would be in any other `express` app!